debian/silkit-utils.silkit-registry.service

```diff
@@ -7,4 +7,6 @@ WantedBy=default.target
 [Service]
 Type=simple
 ExecStart=/usr/bin/sil-kit-registry -s
+ReadOnlyDirectories=/home
```

Comment:
technically, we only need access to `/tmp`. could we disable filesystem access to anything else?

Reply:
We could set 

- ProtectSystem: set to strict -> everything is read only except /dev /proc /sys
- ReadWritePaths -> use this to whitelist /tmp

Reply:
More info: https://www.freedesktop.org/software/systemd/man/latest/systemd.exec.html#Sandboxing

Reply:
Let's do that in a separate PR